packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,7 @@
         <category android:name="android.intent.category.DEFAULT" />
         <category android:name="android.intent.category.BROWSABLE" />
         <data android:scheme="celo" android:host="wallet" />
+        <data android:scheme="wc" android:host="wallet" />
```

Comment:
Are links actually starting with `wc://wallet`?
if not we should remove the `android:host` part.

Reply:
Good point, they're like `wc:123` so no need for that.